chia/full_node/block_store.py

```diff
@@ -60,11 +60,14 @@ async def create(cls, db_wrapper: DBWrapper):
             )
 
             # Height index so we can look up in order of height for sync purposes
-            await self.db.execute("CREATE INDEX IF NOT EXISTS full_block_height on full_blocks(height)")
+            await self.db.execute("CREATE INDEX IF NOT EXISTS height on full_blocks(height)")
```

Comment:
This index may be removed for v2 if height is already indexed in main_chain index.

Reply:
I'm pretty sure we make lookups by height alone as well. but maybe not, in which case you're right

Reply:
If the index has height column on first position, which is the case  for main_chain, than sqlite should also use the other for height lookups.

Reply:
not if the index is constrained `WHERE in_main_chain = 1`. But I forgot whether I did that or not. Either way, I'll review which indices are needed here